pkg/operator/ceph/cluster/mgr/spec_test.go

```diff
@@ -91,6 +92,7 @@ func TestServiceSpec(t *testing.T) {
 		cephv1.DashboardSpec{},
 		v1.ResourceRequirements{},
 		metav1.OwnerReference{},
+		"/var/lib/rook/",
```

Comment:
should we create a const somewhere for `/var/lib/rook` since it's used so widely?

Reply:
I see it used a lot in the tests, and I would rather see tests hardcoded like this. Referencing constants can lead to situations where the tests are just testing exactly what the code is doing instead of what the developer intends the code to do.

Reply:
yes, sounds good for tests